pages/rolam.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
```

Comment:
It's a minor bit weird that the path is `/rolam` but the pciture is `about-me.png` 😄

Reply:
I agree, but the code is English and I would keep the paths in Hungarian as expectedly the most customers will be Hungarian. If I change anything, than it had to be rolam.png. I guess I mean it's doable :)

Reply:
I feel like I always end up being the most opinionated regarding the naming of things, but yes, renaming the picture sounds nice 😄 😄

Reply:
as an aside, all content will come from the CMS in the future. I am saying this because it is not the first time that content-related feedback came and stuff in the code was modified based on it. It is all right and enforces good practices and we all can learn from it, which is second to no other goals in importance. :D Still, let's not forget about this

Reply:
Got no idea why, but this change broke the CSS of the deployment